### PR TITLE
ShowResultsInReadme

### PR DESCRIPTION
**Pull request by MarkusKuemmerle**
Harmonization with content on CAMARA Website

### CODE REVIEW

**Comment by hdamker** [comment]
Updated the version information in "Results" to v0.8.0. 
@shilpa-padgaonkar Should address your comments, please review.
@MarkusKuemmerle Would the content fit for the webpage?

**Review by shilpa-padgaonkar — DISMISSED** [review]

**Review by eric-murray — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -28,8 +28,7 @@ Repository to describe, develop, document and test the QualityOnDemand API famil
 * Version 0.8.0 of the API :
   * API [definition](https://github.com/camaraproject/QualityOnDemand/tree/main/code/API_definitions)
   * API [documentation](https://github.com/camaraproject/QualityOnDemand/tree/main/documentation/API_documentation)
-* There is currently implementation [code](https://github.com/camaraproject/QualityOnDemand/tree/main/code/API_code) available for the initially contributed API version v0.1.0.
-* Going forward provider implementations (PI) will be provided within seperate repositories:
+* Provider implementations (PI) will be provided within seperate repositories:
```
Comment (SfnUser) [inline README.md:31]: seperate -> separate

Comment (hdamker) [reply README.md]: Done

**Comment by hdamker** [comment]
Changed links to point to API definition and documentation in the release branch.

**Review by shilpa-padgaonkar — APPROVED** [review]

**Pull request merged**